app/src/main/java/de/danoeh/antennapod/fragment/DiscoveryFragment.java

```diff
@@ -116,28 +185,40 @@ public void onDestroy() {
         adapter = null;
     }
 
-    private void loadToplist() {
+    private void loadToplist(String country) {
         if (disposable != null) {
             disposable.dispose();
         }
+
         gridView.setVisibility(View.GONE);
         txtvError.setVisibility(View.GONE);
         butRetry.setVisibility(View.GONE);
         txtvEmpty.setVisibility(View.GONE);
         progressBar.setVisibility(View.VISIBLE);
 
-        ItunesTopListLoader loader = new ItunesTopListLoader(getContext());
-        disposable = loader.loadToplist(25).subscribe(podcasts -> {
+        if (country.equals(ItunesTopListLoader.DISCOVER_HIDE_FAKE_COUNTRY_CODE)) {
+            gridView.setVisibility(View.GONE);
+            txtvError.setVisibility(View.VISIBLE);
+            txtvError.setText(String.format(getResources().getString(R.string.discover_is_hidden),
+                    getResources().getString(R.string.discover_hide)));
```

Comment:
Sorry, I noticed another thing. Here, `String.format` is no longer needed.

Reply:
Done